### DISCUSSION TITLE
Support of Azure Stack HCI 23H2+ Logical networks and cluster reserved IPs

### DISCUSSION THREAD

**Discussion by Karl-WE** [Azure/ipam#292] [Ideas]
Good day,

Thank you for this solution. Have not tried it out yet but wonder if it can also help with the Management of network scopes created for Azure Stack HCI 23H2.

Arc > Azure Stack HCI > All clusters > Clusterobject > Resources > Logical Networks

As well as the Cluster internal network specified upon creation (start / end IP), cluster IP.

**Comment by DCMattyG** [comment]
Hi @Karl-WE, this request is exactly why we added the **External Networks** feature and recently expanded that to include **External Subnets** and **External Endpoints**.

Here you can see this in action within the Azure IPAM UI:

![image](https://github.com/Azure/ipam/assets/12955732/e2d392cc-db35-4b66-bf53-def04b3e7c48)

![image](https://github.com/Azure/ipam/assets/12955732/c71463e8-eff1-4ab7-b78b-314b1ec09135)

![image](https://github.com/Azure/ipam/assets/12955732/195b16d1-8ebb-46bc-9bcc-98d83982b758)

Here you're able to create & track networks/subnets/endpoints that live external to Azure, especially for use-cases such as Azure Stack HCI. This is all exposed via API as well if you'd prefer to integrate this process into existing pipelines or CI/CD processes.

Let me know how that look and also happy to help you get started if you require any assistance!

**Reply by Karl-WE** [reply]
Thank you for your explaination and kind offer to help @DCMattyG. It's good to know it is pointing in this direction. I wasn't sure how to understand the external networks feature I have spotted on the release notes. Will try to implement this.
